### DISCUSSION TITLE
November 13th: Clojure/conj 2025 Day 1

### DISCUSSION THREAD

**Discussion by frenchy64** [madclj/madclj.com#20] [Announcements]
👍 to RSVP

https://madclj.com/

Thursday, November 13th, 2025
8:30 AM to 4:00 PM CT

[StartingBlock Madison](https://www.google.com/maps/search/?api=1&query=43.081245%2C%20-89.374054)
821 E Washington Ave 2nd floor · Madison, WI

Day 1 of watch party for https://www.2025.clojure-conj.org/schedule

**Comment by sivakusayan** [comment]
@brunchboy Are you going to Clojure/conj in person this year? If you're coming to the watch party instead and know what days you want to watch, I could visit Madison that same day!

I'll probably come to at least one of the days for the watch party either way. Probably Wednesday so we can watch Eric's talk?

**Reply by brunchboy** [reply]
Alas, neither! We’ll be on our way to Hong Kong early the morning of November 13th, and touring Asia until December 2nd.

**Reply by sivakusayan** [reply]
Ahh, maybe next time 🙂 hope you two have fun!

@frenchy64 were you thinking of holding a watch party on Friday? The two talks at the Friday 10:30AM-11:00AM slots looked interesting, so maybe I could come on both Wednesday and Friday. No big deal if not though.

**Reply by frenchy64** [reply]
@sivakusayan yep, see #29. I split the days up for food purposes.

**Comment by frenchy64** [comment]
@madclj/meetup there'll be meetups on 13th and 14th of November to watch Clojure/conj 2025:

- [RSVP 13th](https://github.com/orgs/madclj/discussions/20)
- [RSVP 14th](https://github.com/orgs/madclj/discussions/29)

**Reply by brunchboy** [reply]
Have fun, Clojurian friends! We’ll be en route to Southeast Asia, air travel disruptions permitting.

**Reply by frenchy64** [reply]
Awesome!
